package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ethersphere/swarm-extension",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
I'd suggest to let the CI (release-please) do the version bumps

Reply:
it has to be changed in the manifest.json as well. Can you configure the release-please to do that?

Reply:
I'll have a look into this, but why can't we just bump the manifest.json manually to the expected release version (this can even be done in the release PR), rather than modifying the `package.json` and `package-lock.json` manually here? There could even be a part of checklist just how it is in `bee-js` https://github.com/ethersphere/bee-js/blob/master/.github/config.yaml#L4-L23

Reply:
you are right, I merge this PR then

Reply:
can you setup the release-please to continue from the version where we are currently?